src/lib/y2partitioner/dialogs/partition_type.rb

```diff
@@ -0,0 +1,69 @@
+require "yast"
+require "ui/event_dispatcher"
+
+module Y2Partitioner
+  module Dialogs
+    # Formerly MiniWorkflowStepPartitionType
+    # FIXME: InnerDialog?!
+    class PartitionType
+      include Yast::UIShortcuts
+      include Yast::Logger
+      include Yast::I18n
+
+      include UI::EventDispatcher
```

Comment:
After talking with @jreidinger I realized that basing this on EventDispatcher and http://www.rubydoc.info/github/yast/yast-yast2/UI%2FDialog is bad because they deal with a plain UI loop whereas we want CWM widgets. I will create a CWM Dialog class.